examples/apo.rs

```diff
@@ -0,0 +1,348 @@
+//! Sighash any-prevout emulation using the new opcodes.
+
+use bitcoin;
+use elements::confidential::{Asset, Value};
+use elements::encode::{self, deserialize};
+use elements::hashes::hex::{FromHex, ToHex};
+use elements::{confidential, opcodes, AddressParams, AssetId, TxOut};
+use miniscript::descriptor::Tr;
+use miniscript::extensions::{
+    AssetExpr, CovExtArgs, CovOps, ParseableExt, Spk, SpkExpr, ValueExpr,
+};
+use miniscript::miniscript::satisfy::{Satisfaction, Witness};
+use miniscript::miniscript::types::extra_props::{OpLimits, TimelockInfo};
+use miniscript::miniscript::types::{Correctness, ExtData, Malleability};
+use miniscript::{expression, Extension, TxEnv};
+extern crate elements_miniscript as miniscript;
+
+use std::fmt;
+
+/// The data that needs to be signed in apo + all.
+/// We can decompose this into into individual parts for fixing version, amt, script pubkey
+///
+/// This structure is onyl an example of how one might implement extension. We do pay any
```

Comment:
In 66aeb41968c171817a93fc30ae00471f6a8dbe90:

nit `onyl` should be `only`

Reply:
Also `do pay` should be `do not pay` I think :)

Reply:
Lol :)